disko-raid1-uefi.nix

```diff
@@ -53,9 +45,12 @@
       type = "mdadm";
       level = 1;
       content = {
-        type = "filesystem";
-        format = "ext4";
-        mountpoint = "/";
+        type = "gpt";
+        partitions.primary = {
```

Comment:
Upstream config uses `partitions.primary`. Will this break stuff?

Reply:
Probably that's what disko requires now, since you've bumped it should be good :crossed_fingers: